mobile/src/main/java/ch/epfl/sdp/mobile/application/social/SocialFacade.kt

```diff
@@ -28,4 +31,13 @@ class SocialFacade(private val auth: Auth, private val store: Store) {
       it.mapNotNull { doc -> doc?.toProfile() }
     }
   }
+  /**
+   * Follows the given user
```

Comment:
Can you be more specific? What this call does when it's called? Which values will be updated ?

I think you can explain why the update is not in the current user's document, but the followed one. It's something that is not trivial if you don't work often with firebase

Reply:
Yes, this was actually my initial comment but I changed that to make it shorter. Will re-elaborate on that, thanks!